File: src/main/java/com/github/prasyloyola/familytree/model/RelationshipWithDetails.java
package com.github.prasyloyola.familytree.model;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;

@Data
@Entity
public class RelationshipWithDetails {

    @Id
    Integer id;
    String relationship;
    String p1_name;
    String p1_gender;
    Integer p1_id;
    String p1_stories;
    String p2_name;
    String p2_gender;
    Integer p2_id;
    String p2_stories;


}

File: src/main/js/application/family-tree-ui/src/app/relationship-with-details.ts
export class RelationshipWithDetails {
    id: number;
    relationship: string;
    p1_name: string;
    p1_gender: string;
    p1_id: number;
    p1_stories: string;
    p2_name: string;
    p2_gender: string;
    p2_id: number;
    p2_stories: string;

}


export class RelationshipWithDetailsResp {
    relationshipWithDetailses : RelationshipWithDetails[]
}